src/components/AddForm.js

```diff
@@ -64,8 +66,14 @@ export class AddForm extends Component {
       tagMoreWater: false,
       tagLittleSun: false,
       tagLittleWater: false,
+      isPlantAdded: true,
+    });
+  };
+
+  handleCloseInfo = () => {
+    this.setState({
+      isPlantAdded: false,
```

Comment:
in this place I put "id: plant.id" for sending it to PlantAddedInfo Component, but it did not work, so I delete it.